src/data_c.c4

```diff
@@ -700,6 +708,52 @@ shmem_signal_fetch(const uint64_t* sig_addr)
     return val;
 }
 
+void SHMEM_FUNCTION_ATTRIBUTES
+shmemx_signal_add(uint64_t *sig_addr, uint64_t signal, int pe)
```

Comment:
Should sig_addr be **const**? I know [in the PR where this was added in the spec](https://github.com/openshmem-org/specification/pull/489/files) the change was suggested to remove the **const** modifier from **sig_addr**, but it looks like [in the main branch of the OpenSHMEM spec repo](https://github.com/openshmem-org/specification/blob/master/content/shmem_signal_add.tex) it is still a **const** parameter